src/css/httpobs.css

```
@@ -386,6 +386,10 @@ sup, sub {
     font-weight: bold;
 }
 
+#scoring-banner:before {
+    content: '🎉';
```

A red emoji on a green background looks truly awful to me, I imagine it's my slight red-green colour blindness, I'd suggest keeping the existing monochromatic emoji:

```suggestion
    content: '📣\FE0E';
```